font-src/glyphs/letter/greek/lower-epsilon.ptl

```diff
@@ -234,6 +307,6 @@ glyph-block Letter-Greek-Lower-Epsilon : begin
 		local dimUpper : CyrZeShapeDim         CAP            midGap blend
 		local dimLower : SmallEpsilonShapeDim  (CAP - midGap) 0      blend
 		include : union
-			CyrZeUpperShape        SLAB-NONE  CAP            midGap blend Hook
-			SmallEpsilonLowerShape SLAB-NONE (CAP - midGap) 0      blend Hook
+			CyrZeUpperShape        SLAB-NONE  CAP           midGap SB RightSB blend Hook
+			SmallEpsilonLowerShape SLAB-NONE (CAP - midGap) 0      SB RightSB blend Hook
```

Comment:
This corrupts U+1D24 actually.
No actions for you. I fixed it by myself in later commits.